README.md

```diff
@@ -1 +1,27 @@
-# Hello-Microverse
\ No newline at end of file
+
+Project Name
+Making Hello Micorverse Project
+
+Built With
+-HTML
+-CSS
+
+Getting Started:
+This is simple project from microverse how to setup a repo in your local machine.
+
+👤 Author
+
+GitHub: @basitali111
+Twitter: @BasitAl35031734
+LinkedIn: BasitAl35031734
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+
+📝 License
+This project is MIT licensed.
```

Comment:
- [ ] The previous reviewers mentioned that the README is supposed to look familiar to [this](https://gist.github.com/lmcneel/45594e550a3403d589bdcaad38138a83) README. Please implement it ass it is a core requirement to pass this milestone. Currently, it does not resemble that as seen in the [template](https://gist.github.com/lmcneel/45594e550a3403d589bdcaad38138a83) to follow 👍